sky/shell/platform/ios/sky_surface.mm

```diff
@@ -179,6 +185,13 @@ - (void)keyboardWillBeHidden:(NSNotification*)notification {
   _engine->OnViewportMetricsChanged(_viewportMetrics.Clone());
 }
 
+- (void)onLocaleUpdated:(NSNotification*)notification {
+    NSLocale *currentLocale = [NSLocale currentLocale];
```

Comment:
Me neither. @tvolkert Its a good idea to run clang-format using the default engine rules. I have it hooked up to my editor.